lib/pushmi_pullyu/swift_depositer.rb

```diff
@@ -16,30 +14,18 @@ def initialize(connection, container)
       authtenant_name: connection[:tenant],
       service_type: 'object-store'
     )
-
-    @swift_container = container
   end
 
-  def deposit_file(file_name)
-    # check if file exists
-    raise "File #{file_name} does not exist" unless File.file?(file_name)
+  def deposit_file(file_name, swift_container)
```

Comment:
Moved the container argument into the deposit_file method. This gives a bit more flexibility in case you wanted to deposit files into different containers. You can then use the same SwiftDepositer connection instance instead of creating multiple SwiftDepositer connections for this.